rados/rados_test.go

```diff
@@ -296,6 +343,54 @@ func (suite *RadosTestSuite) TestMakeDeletePool() {
 	if found {
 		suite.T().Error("Deleted pool still exists")
 	}
+
+	pool, _ = suite.conn.GetPoolByName(new_name)
+	if pool != 0 {
+		suite.T().Error("Pool should have been deleted, but was found!")
+		return
+	}
+}
+
+func (suite *RadosTestSuite) TestGetPoolByID() {
+	suite.SetupConnection()
+
+	// check that new pool name is unique
+	new_name := uuid.Must(uuid.NewV4()).String()
+	pool, err := suite.conn.GetPoolByName(new_name)
+	if pool != 0 || err == nil {
+		suite.T().Error("Random pool name exists!")
+		return
+	}
+
+	// create pool
+	err = suite.conn.MakePool(new_name)
+	assert.NoError(suite.T(), err)
+
+	// verify that the new pool name exists
+	id, err := suite.conn.GetPoolByName(new_name)
+	assert.NoError(suite.T(), err)
+
+	if id == 0 {
+		suite.T().Error("Cannot find newly created pool")
+	}
+
+	// get the name of the pool
+	name, err := suite.conn.GetPoolByID(id)
+	assert.NoError(suite.T(), err)
+
+	if name == "" {
+		suite.T().Error("Cannot find name of newly created pool")
+	}
+
+	// delete the pool
+	err = suite.conn.DeletePool(new_name)
+	assert.NoError(suite.T(), err)
+
+	// verify that it is gone
+	pool, _ = suite.conn.GetPoolByName(new_name)
```

Comment:
`GetPoolByID`?

Reply:
assert for `err`

Reply:
Yeah, GetPoolByID() would be better.
An error is expected, the pool should not exist anymore.